src/components/widgets/ColorWidget.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from "react";
+
+
+function ColorWidget({
+  schema,
+  id,
+  placeholder,
+  value,
+  required,
+  onChange
+}) {
+  return (
+    <input type="color"
+      id={id}
+      className="form-control"
+      value={typeof value === "undefined" ? "" : value}
+      placeholder={placeholder}
+      required={required}
+      onChange={(event) => onChange(event.target.value)} />
+  );
+}
+
+if (process.env.NODE_ENV !== "production") {
+  ColorWidget.propTypes = {
+    schema: PropTypes.object.isRequired,
+    id: PropTypes.string.isRequired,
+    placeholder: PropTypes.string,
```

Comment:
I don't think this input supports placeholders, we should drop support for this prop for this widget.